apypie/api.py

```diff
@@ -156,22 +156,20 @@ def call(self, resource_name, action_name, params={}, headers={}, options={}):
         if not options.get('skip_validation', False):
             action.validate(params)
 
-        return self._call_action(action, params, headers, options)
+        return self._call_action(action, params, headers, options, **kwargs)
 
-    def _call_action(self, action, params={}, headers={}, options={}):
+    def _call_action(self, action, params={}, headers={}, options={}, **kwargs):
         route = action.find_route(params)
         get_params = dict((key, value) for key, value in params.items() if key not in route.params_in_path)
         return self.http_call(
             route.method,
             route.path_with_params(params),
             get_params,
-            headers, options)
+            headers, options, **kwargs)
 
-    def http_call(self, http_method, path, params=None, headers=None, options=None):
+    def http_call(self, http_method, path, params=None, headers=None, options=None, **kwargs):
         full_path = urljoin(self.uri, path)
-        kwargs = {
-            'verify': self._session.verify,
-        }
+        kwargs.update({'verify': self._session.verify})
```

Comment:
Why do we do this? We already set `verify` on the session so that sounds redundant.

Reply:
https://github.com/Apipie/apypie/pull/25

Reply:
Perhaps good to add a comment about this :) I'm also wondering if we should update kwargs or instantiate a new dict and update that.